core-library/src/main/java/org/silverpeas/core/admin/domain/DomainDriver.java

```diff
@@ -24,6 +24,7 @@
 package org.silverpeas.core.admin.domain;
 
 import org.silverpeas.core.admin.domain.model.DomainProperty;
+import org.silverpeas.core.admin.quota.exception.QuotaException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.admin.quota.exception.QuotaException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)